index.html

```diff
@@ -9,8 +9,12 @@
   <link rel="stylesheet" type="text/css" href="scss/application.css">
 </head>
 <body>
+  <div class="navbar">
+    <img class="navbar-logo" src="./assets/wolox_logo.svg" alt="Wolox Logo">
+    <h1 class="title">Developer Tools</h1>
```

Comment:
Remember there is only 1 `h1` per screen / section. I would use `h2` in this case